### PR TITLE
Fix for Issue 574

### PR DESCRIPTION
**Pull request by rachelrj**
Most of the updates were on our side to allow for the impressions to be recorded. Also needed this update to render the creative correctly. I think this should be it.

### CODE REVIEW

File: adapters/sovrn/sovrn.go [hunk 1]
```diff
@@ -250,6 +250,7 @@ func (s *SovrnAdapter) MakeBids(internalRequest *openrtb.BidRequest, externalReq
 	for _, sb := range bidResp.SeatBid {
 		for i := 0; i < len(sb.Bid); i++ {
 			bid := sb.Bid[i]
+			bid.AdM, _ = url.QueryUnescape(bid.AdM)
```
Comment (dbemiller) [inline adapters/sovrn/sovrn.go]: No error handling here?

Should this be an `adapters.BadServerResponeError`?

Comment (rachelrj) [reply adapters/sovrn/sovrn.go]: Good point. But.. I'd rather just drop that specific bid than error on the entire response.

Comment (rachelrj) [reply adapters/sovrn/sovrn.go]: That work?

Comment (dbemiller) [reply adapters/sovrn/sovrn.go]: It's ok with me.

May I ask why you wouldn't want to use the `BadServerResponseError`? I don't see any drawbacks... but there are definitely a few benefits.

To be clear: your bids will still count if you return bids _and_ errors. These just help debugging & diagnosing errors..

Comment (dbemiller) [reply adapters/sovrn/sovrn.go]: Merged... but also feel free to submit another PR adding the error if it changes your mind.

**Pull request merged**